calicoctl/commands/config.go

```diff
@@ -108,7 +108,7 @@ The table below details the valid config options.
 	cf := parsedArgs["--config"].(string)
 	client, err := clientmgr.NewClient(cf)
 	if err != nil {
-		fmt.Println(err)
+		fmt.Printf("Syntax error in %s: %s\n", cf, err)
```

Comment:
I don't think this is right - `clientmgr.NewClient` can return errors that aren't syntax errors (e.g. [this branch](https://github.com/projectcalico/calicoctl/pull/1670/files#diff-bd7ad3177a0a66cbe8ca523f2bb2f8efL31)).

I think this should not have the "Syntax error" bit - same for resources.go